hibernate-core/src/main/java/org/hibernate/bytecode/enhance/spi/interceptor/LazyAttributesMetadata.java

```diff
@@ -35,7 +37,8 @@ public static LazyAttributesMetadata from(
 			PersistentClass mappedEntity,
 			boolean isEnhanced,
 			boolean allowEnhancementAsProxy,
-			boolean collectionsInDefaultFetchGroupEnabled) {
+			boolean collectionsInDefaultFetchGroupEnabled,
+			PersisterCreationContext creationContext) {
```

Comment:
I'm assuming that changing this SPI is OK in master.

Note to self: when this gets backported, the old version of the method should be deprecated.

Reply:
Again, I cannot see this being a method users will call.  This is specifically creating these objects